server.js

```diff
@@ -24,6 +24,7 @@ const pool = mariadb.createPool({
 const PORT = process.env.PORT || 8080;
 const WEBSITE_ACCESS_TOKEN = process.env.WEBSITE_ACCESS_TOKEN;
 const HERALD_TOKEN = process.env.HERALD_TOKEN;
+const LOCALE = {longitude: process.env.LONGITUDE, latitude: process.env.LATITUDE};
```

Comment:
Please abbreviate the JS variables to `lat` and `long`, respectively, just for brevity's sake. No problem keeping the env vars as the full word—that's your choice.